requirements.txt

```
@@ -1,7 +1,7 @@
 black==22.3.0
 flake8==5.0.4
 flake8-tidy-imports==4.6.0
-graviton@git+https://github.com/algorand/graviton@v0.5.0
+graviton@git+https://github.com/algorand/graviton@26d47e31c3ffb5132cd29251eefcb50db0723d03
```

```suggestion
graviton@git+https://github.com/algorand/graviton@v0.7.0
```

Gonna bump up the graviton version because of breaking changes: https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase